source/manual/sentry.html.md

```diff
@@ -7,12 +7,215 @@ section: Monitoring
+[data_sync_excluded_exceptions]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configure.rb#L57-L65
+[chain-inspection]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configuration.rb#L43
+
+### Ignoring environments
+
+There is a hardcoded list of environments in which Sentry is
+considered to be active. If the `SENTRY_CURRENT_ENV` environment variable
+available to your app is not on the
+[`active_sentry_environments`][active_sentry_environments] list, then none of
+its errors will be logged to Sentry.
+
+[active_sentry_environments]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configure.rb#L14-L18
+
+### Advanced Sentry customisation
+
+For all of the above, you can easily
+[configure any of these properties by appending to them][error-configuration].
+
+You can also run arbitrary code to decide whether or not an error should be
+logged in Sentry, using the `should_capture` lambda. Your custom code will be
+[lazily combined with the default evaluators][combined-code] such that if an
+exception is on any of the 'excluded exceptions' lists, it will be excluded
+(even if your custom `should_capture` callback returns true).
+
+Find out more in the [govuk_app_config README][govuk_app_config-readme].
+
+[error-configuration]: https://github.com/alphagov/govuk_app_config/blob/master/README.md#error-configuration
+[combined-code]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configuration.rb#L18-L24
+[govuk_app_config-readme]: https://github.com/alphagov/govuk_app_config/blob/master/README.md#error-reporting
+
+## When errors are received by Sentry
+
+Sentry first fingerprints the error to decide what Issue to group it under.
+It then checks how many occurrences of that Issue have happened recently,
+and rate-limits (i.e. ignores) this occurrence if it is happening too
+frequently. More details below.
+
+### Fingerprinting
+
+Errors are grouped into Issues automatically by Sentry. By default, Sentry
+[groups based on stack trace][stack-trace-grouping]. If no stack trace is
+available, it will [group by exception type][exception-grouping] instead.
+Failing that, it will group by error message.
+
+Fingerprint rules can be applied at the project level by editing the
+"Issue Grouping" page under a project (see [example][example-issue-grouping]).
+For example, you can force all errors of the same exception type to have the
+same fingerprint. On the same screen, you can set stack trace rules, to
+remove or rename certain 'frames' in the trace. In practice, we don't
+currently apply any custom rules, after an in-depth [investigaton][]
```

Comment:
is there an investigation link for here?

Reply:
Yes - `[]` refers to the link text as its alias, which is `investigation`, defined here: https://github.com/alphagov/govuk-developer-docs/pull/2927/files#diff-f59e86b993e1bbe8d27722a2a604d3df960a1282ede39f6f89fa6bc3fc96315cR199

Reply:
ah, it's another typo - sorry i spotted the link wasn't working but not the reason why.
`investigaton` => `investigation`

Reply:
Ahhhhh good spot! 👏